ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -658,15 +668,16 @@ double UncoupledProcessesTimeLoop::computeTimeStepping(
         }
         else
         {
-            if (std::abs(dt -prev_dt) < std::numeric_limits<double>::min()
-                && _last_step_rejected)
+            if (std::abs(dt - prev_dt) < std::numeric_limits<double>::min() &&
```

Comment:
I think this will only be true if `dt == prev_dt`, because AFAIK std::numeric_limits<double>::min() is 1e-300 or so.